drivers/spi/Kconfig.xec_qmspi

```diff
@@ -0,0 +1,12 @@
+# Kconfig - Atmel SAM0 SERCOM SPI
```

Comment:
Please change the description (or remove it if you want).

Reply:
Done.